### PR TITLE
Improve analyze_coincidences.py script + add small corrections

### PR DESCRIPTION
**Pull request by flukson**

### CODE REVIEW

File: goja/goja_manager.py [hunk 1]
```diff
@@ -23,91 +26,152 @@ def run_simulation(path_gate_output, type_of_run):
...
+      array_pbs.write('#PBS -N GOJA' + ss + '\n')
+      array_pbs.write('#PBS -V\n')
+      array_pbs.write('cd ${PBS_O_WORKDIR}\n')
+      array_pbs.write(goja_command + '\n')
+      array_pbs.write('exit 0;\n')
+    # push into queue:
+    qsub_command = 'qsub ' + ARRAY_GOJA_PBS
+    os.system(qsub_command)
+    # remove ARRAY_GOJA_PBS:
+    os.unlink(ARRAY_GOJA_PBS)
```
Comment (graag) [inline goja/goja_manager.py:77]: Just curious. Isn't it usefull to check what parameters were used for particular job? Or is it stored somewhere?

File: goja/goja_manager.py [hunk 2]
```diff
@@ -23,91 +26,152 @@ def run_simulation(path_gate_output, type_of_run):
...
+    missing_gate_results = missing_gate_results_txt.readlines()
+    if len(missing_gate_results)>0:
+      for m in missing_gate_results:
+        m = m.replace('\n', '')
+        with open("array.pbs", "r") as array_pbs:
+          with open("array.pbs.tmp", "w") as array_pbs_tmp:
+            for line in array_pbs:
+              array_pbs_tmp.write(line.replace('${PBS_ARRAYID}', m))
+        os.system('qsub array.pbs.tmp')
+        os.unlink('array.pbs.tmp')
```
Comment (graag) [inline goja/goja_manager.py:42]: Just curious. Isn't it usefull to check what parameters were used for particular job? Or is it stored somewhere?

File: goja/goja_manager.py [hunk 3]
```diff
@@ -407,14 +423,21 @@ def concatenate_files(fnames):
...
-    command = 'rm -f ' + args.path_goja_output + '*'
+    print "Clear (GOJA):"
+    command = 'rm -f ' + path_goja_output + '*'
+    print '\t' + command
+    os.system(command)
+
+  elif args.mode == "clear-cluster-artifacts":
+
+    print "Clear (cluster artifacts):"
+    command = 'rm -f *.o* *.e*'
```
Comment (graag) [inline goja/goja_manager.py:440]: This could be used to remove the scripts if needed instead immediately after qsub.

**Comment by wkrzemien** [comment]
@flukson what is the status of this PR? Is it waiting for the review?

File: goja/goja_manager.py (hunk 1 above)
Comment (flukson) [reply goja/goja_manager.py:77]: I have moved removing the file to clear-cluster-artifacts mode.

File: goja/goja_manager.py (hunk 2 above)
Comment (flukson) [reply goja/goja_manager.py:42]: I have moved removing the file to clear-cluster-artifacts mode.

File: goja/goja_manager.py (hunk 3 above)
Comment (flukson) [reply goja/goja_manager.py:440]: Done.

**Comment by flukson** [comment]
@wkrzemien @graag I have implemented Konrad's comments. I also believe that the code should be refactored in the future and I could work on that if I will continue cooperation with the J-PET project. I think that, for now, this PR could be just merged.

**Review by wkrzemien — APPROVED** [review]

**Pull request merged**